config/locales/en.yml

```diff
@@ -323,23 +325,23 @@ en:
     unsubmitted: Unsubmitted Courses
     untrained: Untrained
     uploads_none: This class has not contributed any images or other media files to Wikimedia Commons.
-    weeks_none: This course does not have a timeline yet.
+    view_other: View other cohorts
     view: View Course
     view_doc: The sum of all views to articles edited by enrolled editors since the editor made their first edit to each article
     view_page: View Course Page
     view_wiki: View on Wikipedia
     volunteer:
       one: Volunteer
       other: Volunteers
+    weeks_none: This course does not have a timeline yet.
     warn_mirrored: >
       Upon submission, this course will be automatically posted to Wikipedia with your user account.
       After that, new edits to the timeline will be mirrored to Wikipedia.
 
       Are you sure you want to do this?
     wiki_staff: Wiki Ed Staff
-    word_count_doc: An estimate of the number of words added to mainspace articles by the course's students during the course term
+    word_count_doc: An estimate of the number of words added to mainspace articles by the program's editors during the program term
```

Comment:
The programs version should be in courses_generic rather than replacing the course version.